easybuild/easyconfigs/p/PheWeb/PheWeb-1.1.20-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,85 @@
+easyblock = 'PythonBundle'
+
+name = 'PheWeb'
+version = '1.1.20'
+versionsuffix = '-Python-%(pyver)s'
+
+
+homepage = 'https://github.com/statgen/pheweb'
+description = """A tool for building PheWAS websites from association files"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Pysam', '0.15.3'),
+    ('openpyxl', '3.0.3', versionsuffix, ('GCCcore', '8.3.0')),
+    ('Flask', '1.1.2', versionsuffix, ('GCCcore', '8.3.0')),
+    ('tqdm', '4.41.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('Brotli', '1.0.7', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['0538dc1744fd17c314d2adc409ea7d1b779783b89fd95bcfb0c2acc93a6ea5a7'],
+    }),
+    ('Flask-Compress', '1.5.0', {
+        'checksums': ['f367b2b46003dd62be34f7fb1379938032656dca56377a9bc90e7188e4289a7c'],
+    }),
+    ('Flask-Login', '0.5.0', {
+        'checksums': ['6d33aef15b5bcead780acc339464aae8a6e28f13c90d8b1cf9de8b549d1c0b4b'],
+    }),
+    ('rauth', '0.7.3', {
+        'checksums': ['524cdbc1c28560eacfc9a9d40c59525eb8d00fdf07fbad86107ea24411477b0a'],
+    }),
+    ('marisa-trie', '0.7.5', {
+        'checksums': ['c73bc25d868e8c4ea7aa7f1e19892db07bba2463351269b05340ccfa06eb2baf'],
+    }),
+    ('sortedcontainers', '2.2.2', {
+        'checksums': ['4e73a757831fc3ca4de2859c422564239a31d8213d09a2a666e375807034d2ba'],
+    }),
+    ('intervaltree', '3.0.2', {
+        'checksums': ['cb4f61c81dcb4fea6c09903f3599015a83c9bdad1f0bbd232495e6681e19e273'],
+    }),
+    ('requests', '2.24.0', {
+        'checksums': ['b3559a131db72c33ee969480840fff4bb6dd111de7dd27c8ee1f820f4f00231b'],
+    }),
```

Comment:
```suggestion
```

`intervaltree` is in Python/3.7.4 and sufficient `sortedcontainers` (2.1.0) and `requests` (2.22.0) are also there